src/patternfly/components/Hero/hero.scss

```diff
@@ -1,43 +1,59 @@
 @use '../../sass-utilities' as *;
 
 @include pf-root($hero) {
+  --#{$hero}--PaddingBlockStart: var(--pf-t--global--spacer--xl);
+  --#{$hero}--PaddingBlockEnd: var(--pf-t--global--spacer--xl);
+  --#{$hero}--PaddingInlineStart: var(--pf-t--global--spacer--3xl);
+  --#{$hero}--PaddingInlineEnd: 0;
```

Comment:
I think we'll want a `PaddingInlineEnd`, probably set to `var(--pf-t--global--spacer--xl)` since not everyone will use `__body` and they may just put title/text content in there. 

<img width="888" height="320" alt="Screenshot 2025-11-07 at 1 43 21 PM" src="https://github.com/user-attachments/assets/45a73eb4-0bcc-4310-a621-f57af765d93a" />